modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeJavaCheck.java

```diff
@@ -54,15 +55,28 @@ private String _fixImports(JavaClass javaClass, String content)
 			String newImportName = importsMap.get(importName);
 
 			if (newImportName != null) {
-				content = StringUtil.replace(
-					content, StringBundler.concat("import ", importName, ";"),
-					StringBundler.concat("import ", newImportName, ";"));
+				if (_isClassNameEquals(importName, newImportName)) {
+					content = _replaceImport(
+						content, importName, newImportName);
+				}
+				else {
+					content = _replaceImport(
+						content, importName, newImportName);
+
+					content = _replaceImpl(content, _className, _newClassName);
```

Comment:
@michaelmdelcavalcanti, the second and third parameters of this method are actually attributes from this class, in light of this why don't we invoke them directly from the method implementation and keep just the first parameter?